Admin/MediaAdminExtension.php

```diff
@@ -11,19 +11,21 @@
 
 namespace CoopTilleuls\Bundle\CKEditorSonataMediaBundle\Admin;
 
-use Sonata\AdminBundle\Admin\AdminExtension;
+use Sonata\AdminBundle\Admin\AbstractAdminExtension;
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Sonata\AdminBundle\Route\RouteCollection;
 
 /**
  * Adds browser and upload routes to the Admin
  *
  * @author Kévin Dunglas <kevin@les-tilleuls.coop>
  */
-class MediaAdminExtension extends AdminExtension
+class MediaAdminExtension extends AbstractAdminExtension
```

Comment:
So we need to drop Sonata 2.3 from https://github.com/coopTilleuls/CoopTilleulsCKEditorSonataMediaBundle/blob/master/composer.json#L16